src/ebonite/build/docker.py

```diff
@@ -44,11 +42,8 @@ def get_host(self) -> str:
 class RemoteDockerRegistry(DockerRegistry):
     type = 'remote'
 
-    def __init__(self, host: str):
-        if re.match(VALID_HOST_REGEX, host):
-            self.host = host
-        else:
-            raise ValueError('Host {} is not valid'.format(host))
+    def __init__(self, host: Optional[str] = None):
+        self.host = host or 'https://index.docker.io/v1/'
```

Comment:
why is this needed?

Reply:
This is an address of DockerHub's registry. I think it's good to specify it as default here not to require our users to Google for it (strangely enough it is not so easy to find).

Reply:
yes, but i thought that it was default for docker - if you dont set it, docker registry is used, no?

Reply:
In earlier version of this class we were required to specify host. Simply putting `None` caused and still causes exceptions in several parts of our code (at least, `DockerImage.get_uri` and `login_to_registry`). Moreover, Docker client `login` method is kind of cryptic here: it has `registry=None` but its documentation says "registry (str): URL to the registry.  E.g. ``https://index.docker.io/v1/``". Considering all of this I decided to specify this URL as a safety net.